Whitepaper.tex

```diff
@@ -309,8 +340,8 @@ \subsection{Domain Specific Languages}
 
 \section{Overview}\label{overview}
 \begin{figure}[H]
-    \includegraphics[width=0.49\textwidth]{media/NormalKernelOverview.pdf}
-    \includegraphics[width=0.49\textwidth]{media/BeakerKernelOverview.pdf}
+    \includegraphics[width=0.49\textwidth]{media/NormalKernelOverview}
```

Comment:
Unfortunately it seems that some latex editors do not accept `.svg` files, which is why we had to use inkscape to generate `.pdf`'s. I don't know if there's any latex versions that do support `.svg`'s though.